client/post-editor/media-modal/detail/detail-preview-image.jsx

```diff
@@ -15,7 +16,14 @@ export default React.createClass( {
 
 	propTypes: {
 		site: PropTypes.object,
-		item: PropTypes.object.isRequired
+		item: PropTypes.object.isRequired,
+		onLoad: PropTypes.func,
```

Comment:
If we end up not using it in this PR, lets remove it. We can always add it again later